source/_integrations/tplink.markdown

```diff
@@ -90,19 +92,40 @@ See [Supported Devices in python-kasa](https://github.com/python-kasa/python-kas
 - KL130
 - KL135
 
-### Light strips
+#### Light strips
 
 - KL400
 - KL420
 - KL430
 
-Other bulbs may also work, but with limited color temperature range (2700-5000). If you find a bulb isn't reaching the full-color temperature boundaries, submit a bug report to [python-kasa](https://github.com/python-kasa/python-kasa).
+### Requiring authentication
+
+#### Plugs
+
+- EP25 (Hardware version >= 2.6)
+- KP125M
+- P110
+- HS100 (UK Hardware version 4.1 with firmware 1.1.0)
+
+#### Wall switches
+
+- KS205
+- KS225
+
+#### Bulbs
+
+- L510B
+- L530E
+
+#### Light strips
+
+- L900-5
+- L900-10
```

Comment:
```suggestion
- L900-10
- L920
```

<img width="644" alt="Screenshot 2024-01-22 at 11 34 55 AM" src="https://github.com/home-assistant/home-assistant.io/assets/663432/4c1cffec-63ac-4e63-a1fd-a1063b37fab3">


confirmed working..  I think I added a fixture for this one as well already

Reply:
https://github.com/python-kasa/python-kasa/blob/master/kasa/tests/fixtures/smart/L920-5(US)_1.0_1.1.0.json